src/EditorFeatures/VisualBasicTest/Utils.vb

```diff
@@ -112,8 +112,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UnitTests
             Private ReadOnly _node As T
 
             Sub New(syntaxTree As SyntaxTree, node As T)
+#If Not CODE_STYLE Then
```

Comment:
Why?  Can we not also make these simple helpers available?

Reply:
There is type name conflict in Code style test projects due to Contract being available from both CodeStyle analyzer project and Workspaces project. We can potentially work around this with some IVT massaging, but given the IDE test framework in CodeStyle layer is a temporary workaround, I'd rather just leave this for it to be reverted later.